xml/System.Net.Quic/QuicListenerOptions.xml

```diff
@@ -92,9 +93,9 @@
         <ReturnType>System.Func&lt;System.Net.Quic.QuicConnection,System.Net.Security.SslClientHelloInfo,System.Threading.CancellationToken,System.Threading.Tasks.ValueTask&lt;System.Net.Quic.QuicServerConnectionOptions&gt;&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets or sets the selection callback to choose inbound connection options dynamically.</summary>
+        <summary>Gets or sets the selection callback to choose inbound connection options.</summary>
         <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <remarks>This property is mandatory, not setting it will result in validation errors when starting the listener.</remarks>
```

Comment:
```suggestion
        <remarks>This property is mandatory. Not setting it will result in validation errors when starting the listener.</remarks>
```